.design-sync/ds-tailwind-input.css

```
@@ -17,5 +17,6 @@
 /* Generate utilities for the atoms, foundation specimens, primitives, and previews. */
 @source "../src/features/shared/components/recipe/*.tsx";
 @source "../src/features/shared/components/design-system/*.tsx";
+@source "../src/features/shared/components/Stamp.tsx";
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)